integration_test/test_tools.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quellenreiter_app/main.dart' as app;
+
+class TestTools {
+  /// Helper to pump and wait for [n] seconds
+  static Future<void> pumpFramesForNSeconds(
+      WidgetTester _tester, int _seconds) async {
+    await _tester.pumpFrames(
+        const app.QuellenreiterApp(), Duration(seconds: _seconds));
+  }
+
+  /// Define some commonly used [Finder]s
+  static Finder onLoginScreen = find.text("Konto erstellen");
```

Comment:
Can these be found independently from their text? Like instead find the widget by class?
Hardcoding the text here seems very unstable :)

Reply:
The `find.byKey()` is at least a little better, as the widget keys are less likely to be changed in the future